integration/specs/app_spec.ts

```diff
@@ -6,39 +6,39 @@ jest.setTimeout(30000)
 
 const BACKSPACE = "\uE003"
 
-describe("app start", () => {
-  let app: Application
-  const clickWhatsNew = async (app: Application) => {
-    await app.client.waitUntilTextExists("h1", "What's new")
-    await app.client.click("button.primary")
-    await app.client.waitUntilTextExists("h1", "Welcome")
-  }
+async function clickWhatsNew(app: Application) {
+  await app.client.waitUntilTextExists("h1", "What's new")
+  await app.client.click("button.primary")
+  await app.client.waitUntilTextExists("h1", "Welcome")
+}
+
+async function addMinikubeCluster(app: Application) {
+  await app.client.click("div.add-cluster")
+  await app.client.waitUntilTextExists("div", "Select kubeconfig file")
+  await app.client.click("div.Select__control")
+  await app.client.waitUntilTextExists("div", "minikube")
+  await app.client.click("div.minikube")
+  await app.client.click("div.Select__control")
```

Comment:
This is the added line

Reply:
Yeah, I have added this line in my PR. Did you test this locally? And you have more than one context in your config? The CI won't and I think it will still fail...

Reply:
It fixed locally (I have several other contexts). I see that #888 will probably be broken for its own reasons. But this was breaking for some of my other PRs too

Reply:
It wasn't breaking on the CI, except for #888